assets/sass/components/global/_googlesitekit-stepper.scss

```diff
@@ -74,13 +82,22 @@
 		display: flex;
 		font-family: $f-primary;
 		font-size: $fs-title-md;
+		font-weight: $fw-medium;
 		height: 24px;
 		line-height: $lh-title-md;
-		margin: 0;
+
+		@media (min-width: $bp-tablet) {
+			margin: 0;
+		}
 	}
 
 	.googlesitekit-stepper__step-title--upcoming {
 		color: $c-surfaces-on-surface-variant;
+		display: block;
+
+		@media (min-width: $bp-tablet) {
+			display: flex;
+		}
```

Comment:
Could you help me understand the purpose of this change on this specific variant? Is it because of the string length? If so, do you think it'd be wiser to apply it on `.googlesitekit-stepper__step-title` itself instead of a specific variant?

We could make `.googlesitekit-stepper__step-title` use a `display` `block` and an unset `height` by default, but apply a `display` `flex` and `height` `24px` starting from `$bp-tablet`.

On mobile viewports, if the spacing between the step number and the title is an issue, we could reduce the margin in that case.

What do you think?

Reply:
I think this was to fix something after another change I made, but with your suggested tweaks is no longer needed 😆